h5py/_hl/group.py

```diff
@@ -206,6 +218,7 @@ def create_dataset_like(self, name, other, **kwupdate):
         shape and dtype, in which case the provided values take precedence over
         those from `other`.
         """
+        # TODO: track_times, track_order
```

Comment:
Any reason not do to this here?

Reply:
It required some moderate refactoring, but indeed.